client/lib/react-test-env-setup/test/index.js

```diff
@@ -9,7 +9,7 @@ var jsdom = require( 'jsdom' ).jsdom,
  */
 var setupEnvironment = require( '../' );
 
-describe( 'react-test-env-setupEnvironment', function() {
+describe( 'react-test-env-setup', function() {
 	afterEach( function() {
 		delete global.window;
```

Comment:
Probably we should sync after each with what we do in [cleanup function](https://github.com/Automattic/wp-calypso/blob/update/single-test-runner-ms-rtes-pm/client/lib/react-test-env-setup/index.js#L39). There are more global variables removed.